src/main/java/build/buildfarm/cas/CASFileCache.java

```diff
@@ -188,10 +190,14 @@ public synchronized long size() {
     return sizeInBytes;
   }
 
-  public long storageCount() {
+  public long totalEntryCount() {
```

Comment:
`totalEntryCount` sounds like there's component groups to entries: `entryCount` should be fine